### ISSUE TITLE
Misleading error message when forgetting interface body

### ISSUE THREAD

**Issue by DanielSWolf** [microsoft/TypeScript#44020]
# Bug Report

### 🔎 Search Terms

"An interface can only extend an identifier/qualified-name with optional type arguments", interface, body

### 🕗 Version & Regression Information

- This is the behavior in every version I tried, and I reviewed the FAQ for entries about the error message.

### ⏯ Playground Link

[Playground link with relevant code](https://www.typescriptlang.org/play?ts=4.3.0-beta#code/JYOwLgpgTgZghgYwgAgEJwM4QIwB4AqAfMgN4BQylyAthGABYD2AJtgBQBucANgK4QAuZPgCUQjo2DMA3GQC+ZMqEixEKdFgBMBYuSo06TZps49+Q0eMkz5i5dHhJkAEWjAOEZjuQQAHpBBmDDRMHB0AGhCtHWkgA)

### 💻 Code

The following code tries to declare an interface `Derived<T>` deriving from `Base1<T>` and `Base2<T>` without any extra members. I forgot to type the empty interface body `{}`, instead typing a semicolon.

```ts
interface Base1<T> {
    method1(value: T): void;
}

interface Base2<T> {
    method2(value: T): void;
}

// Missing {} at the end of the following line.
interface Derived<T> extends Base1<T>, Base2<T>;
```

### 🙁 Actual behavior

In the last line of code, I get the following error:

> An interface can only extend an identifier/qualified-name with optional type arguments. (2499)

The error doesn't address the actual problem (the missing braces). Instead, it reads as if there was something wrong with deriving from `Base2<T>`. I actually took it to mean that I cannot derive from a generic interface, so I spent time trying to find out more about this (non-existing) limitation.

### 🙂 Expected behavior

Instead of the misleading error message, I'd expect to get an error telling me that the curly braces are missing.

To be fair, this error *does* show, but not always: If there is a line break after the last line, the correct error will show on the first character of the following empty line (see screenshot). But this is nearly invisible next to the big red (wrong) error above.

![image](https://user-images.githubusercontent.com/5963162/117615015-7191e000-b169-11eb-96e1-9d586ffc9f1c.png)
